src/analysis/normed_space/basic.lean

```diff
@@ -838,17 +859,14 @@ by simpa only [is_unit_iff_ne_zero] using punctured_nhds_ne_bot (0:α)
 end normed_field
 
 instance : normed_field ℝ :=
-{ norm := λ x, abs x,
-  dist_eq := assume x y, rfl,
-  norm_mul' := abs_mul }
+{ norm_mul' := abs_mul,
+  .. real.normed_group }
```

Comment:
As far as I understand, `to_normed_group := real.normed_group` makes it easier for Lean to understand that `real.normed_field.to_normed_group = real.normed_group`.

Reply:
`normed_field` does not extend `normed_group`, so this doesn't work. If we want to refactor this, I think it should be another PR.